vng/writer.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	MaxSegmentThresh = vector.MaxSegmentThresh
+	MaxSegmentThresh = math.MaxInt
```

Comment:
Should we make this change to vector.MaxSegmentThresh intead?

Reply:
Good point.